components/console_cmd_ping/README.md

```diff
@@ -0,0 +1,36 @@
+# Console command ping
+The component provides a console where the 'ping' command can be executed.
+
+## API
+
+### Steps to enable console in an example code:
+1. Add this component to your project using ```idf.py add-dependency``` command.
+2. In the main file of the example, add the following line:
+    ```c
+    #include "console_ping.h"
+    ```
+3. Ensure esp-netif is initialized and default event loop is created in your app_main():
+    ```c
+    ESP_ERROR_CHECK(esp_netif_init());
+    ESP_ERROR_CHECK(esp_event_loop_create_default());
+    ```
+4. In your app_main() function, add the following line as the last line:
+    ```c
+    console_cmd_init();     // Initialize console
+    console_cmd_ping_register();
```

Comment:
For consistency with the block above, could this be like this?
```suggestion
    ESP_ERROR_CHECK(console_cmd_ping_register());
```
(same for other calls)